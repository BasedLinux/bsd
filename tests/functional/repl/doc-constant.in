:l doc-comments.bsd
:doc constant
:doc lib.version
:doc lib.attr.empty
:doc lib.attr.undocument
:doc (import ./doc-comments.bsd).constant
:doc (import ./doc-comments.bsd).lib.version
:doc (import ./doc-comments.bsd).lib.attr.empty
:doc (import ./doc-comments.bsd).lib.attr.undocumented
:doc missing
:doc constanz
:doc missing.attr
:doc lib.missing
:doc lib.missing.attr
:doc lib.attr.undocumental
