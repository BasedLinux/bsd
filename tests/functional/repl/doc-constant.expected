Bsd <bsd version>
Type :? for help.

bsd-repl> :l doc-comments.bsd
Added <number omitted> variables.

bsd-repl> :doc constant
error: value does not have documentation

bsd-repl> :doc lib.version
Attribute `version`

  … defined at /path/to/tests/functional/repl/doc-comments.bsd:47:3


Immovably fixed.

bsd-repl> :doc lib.attr.empty
Attribute `empty`

  … defined at /path/to/tests/functional/repl/doc-comments.bsd:52:3


Unchangeably constant.

bsd-repl> :doc lib.attr.undocument
error:
       … while evaluating the attribute 'attr.undocument'
         at /path/to/tests/functional/repl/doc-comments.bsd:52:3:
           51|   */
           52|   lib.attr.empty = { };
             |   ^
           53|

       error: attribute 'undocument' missing
       at «string»:1:1:
            1| lib.attr.undocument
             | ^
       Did you mean undocumented?

bsd-repl> :doc (import ./doc-comments.bsd).constant
Attribute `constant`

  … defined at /path/to/tests/functional/repl/doc-comments.bsd:42:3


Firmly rigid.

bsd-repl> :doc (import ./doc-comments.bsd).lib.version
Attribute `version`

  … defined at /path/to/tests/functional/repl/doc-comments.bsd:47:3


Immovably fixed.

bsd-repl> :doc (import ./doc-comments.bsd).lib.attr.empty
Attribute `empty`

  … defined at /path/to/tests/functional/repl/doc-comments.bsd:52:3


Unchangeably constant.

bsd-repl> :doc (import ./doc-comments.bsd).lib.attr.undocumented
Attribute `undocumented`

  … defined at /path/to/tests/functional/repl/doc-comments.bsd:54:3

No documentation found.

bsd-repl> :doc missing
error: undefined variable 'missing'
       at «string»:1:1:
            1| missing
             | ^

bsd-repl> :doc constanz
error: undefined variable 'constanz'
       at «string»:1:1:
            1| constanz
             | ^

bsd-repl> :doc missing.attr
error: undefined variable 'missing'
       at «string»:1:1:
            1| missing.attr
             | ^

bsd-repl> :doc lib.missing
error: attribute 'missing' missing
       at «string»:1:1:
            1| lib.missing
             | ^

bsd-repl> :doc lib.missing.attr
error: attribute 'missing' missing
       at «string»:1:1:
            1| lib.missing.attr
             | ^

bsd-repl> :doc lib.attr.undocumental
error:
       … while evaluating the attribute 'attr.undocumental'
         at /path/to/tests/functional/repl/doc-comments.bsd:52:3:
           51|   */
           52|   lib.attr.empty = { };
             |   ^
           53|

       error: attribute 'undocumental' missing
       at «string»:1:1:
            1| lib.attr.undocumental
             | ^
       Did you mean undocumented?
